nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1406,6 +1452,10 @@ class SyncAskResponse(BaseModel):
         title="Answer",
         description="The generative answer to the query",
     )
+    object: Optional[Dict[str, Any]] = Field(
+        title="JSON",
+        description="The generative JSON to the query",
```

Comment:
Can you mention in the description that this is only returned if the `answer_json_schema` parameter is passed in the request?